### ISSUE TITLE
[DynamoDBEnableServerSideEncryption] do not exist in the template 

### ISSUE THREAD

**Issue by monaye** [aws-amplify/amplify-console#35]
**Describe the bug**
I am not sure if this is AmplifyCLI issue or Console issue. I am trying to deploy using Amplify Console and getting the `[DynamoDBEnableServerSideEncryption] do not exist in the template` error. 
It didn't happen until recently and we haven't change the cloud formation template generated by the AmplifyCLI. 
`amplify push` is working fine. 

**To Reproduce**
1. Go to 'Amplify Console'
2. Finish deploy setting 
4. Hit 'deploy' and See error

```
                                 # Starting phase: build
                                 # Executing command: amplifyPush --simple
2019-08-07T16:41:00.824Z [INFO]: # Getting Amplify CLI Cloud-Formation stack info from environment cache
2019-08-07T16:41:00.853Z [INFO]: # Start initializing Amplify environment: dev
2019-08-07T16:41:00.855Z [INFO]: # Initializing new Amplify environment: dev (amplify init)
2019-08-07T16:41:14.816Z [INFO]: Note: It is recommended to run this command from the root of your app directory
2019-08-07T16:41:17.090Z [INFO]: 
2019-08-07T16:41:17.092Z [INFO]: For more information on AWS Profiles, see:
                                 https://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html
2019-08-07T16:41:17.162Z [WARNING]: - Initializing your environment: dev
2019-08-07T16:41:17.708Z [WARNING]: ✔ Initialized provider successfully.
2019-08-07T16:41:24.752Z [INFO]: Could not read cloudformation template at path: /codebuild/output/src150497733/src/handbook/amplify/backend/api/nhb/build/cloudformation-template.json
2019-08-07T16:41:24.863Z [WARNING]: - Updating resources in the cloud. This may take a few minutes...
2019-08-07T16:41:30.488Z [INFO]: 
2019-08-07T16:41:30.492Z [INFO]: UPDATE_IN_PROGRESS nhb-dev-20190802090817 AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:25 GMT+0000 (Coordinated Universal Time) User Initiated
2019-08-07T16:41:30.610Z [INFO]: 
2019-08-07T16:41:30.610Z [INFO]: UPDATE_IN_PROGRESS authnhbcognito AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:30 GMT+0000 (Coordinated Universal Time)
2019-08-07T16:41:35.408Z [INFO]: 
2019-08-07T16:41:35.411Z [INFO]: UPDATE_COMPLETE    authnhbcognito       AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:31 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_IN_PROGRESS apinhb               AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:33 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_IN_PROGRESS functionnhbusersfunc AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:33 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_FAILED      apinhb               AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:33 GMT+0000 (Coordinated Universal Time) Parameters: [DynamoDBEnableServerSideEncryption] do not exist in the template
2019-08-07T16:41:35.593Z [INFO]: 
2019-08-07T16:41:35.594Z [INFO]: UPDATE_IN_PROGRESS          nhb-dev-20190802090817-functionnhbusersfunc-1L6QD27RBKVX AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:33 GMT+0000 (Coordinated Universal Time) User Initiated
                                 UPDATE_ROLLBACK_IN_PROGRESS nhb-dev-20190802090817-functionnhbusersfunc-1L6QD27RBKVX AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:34 GMT+0000 (Coordinated Universal Time) Initiated by parent stack
2019-08-07T16:41:45.410Z [INFO]: 
2019-08-07T16:41:45.412Z [INFO]: UPDATE_FAILED functionnhbusersfunc AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:44 GMT+0000 (Coordinated Universal Time) Resource update cancelled
2019-08-07T16:41:45.715Z [INFO]: 
2019-08-07T16:41:45.716Z [INFO]: UPDATE_ROLLBACK_IN_PROGRESS nhb-dev-20190802090817 AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:45 GMT+0000 (Coordinated Universal Time) The following resource(s) failed to update: [functionnhbusersfunc, apinhb].
2019-08-07T16:41:55.422Z [INFO]: 
2019-08-07T16:41:55.424Z [INFO]: UPDATE_IN_PROGRESS authnhbcognito       AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:51 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_COMPLETE    authnhbcognito       AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:51 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_COMPLETE    apinhb               AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:52 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_IN_PROGRESS functionnhbusersfunc AWS::CloudFormation::Stack Wed Aug 07 2019 16:41:52 GMT+0000 (Coordinated Universal Time)
2019-08-07T16:42:25.483Z [INFO]: 
2019-08-07T16:42:25.485Z [INFO]: UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS nhb-dev-20190802090817-functionnhbusersfunc-1L6QD27RBKVX AWS::CloudFormation::Stack Wed Aug 07 2019 16:42:21 GMT+0000 (Coordinated Universal Time)
2019-08-07T16:42:30.432Z [INFO]: 
2019-08-07T16:42:30.434Z [INFO]: UPDATE_COMPLETE                              functionnhbusersfunc   AWS::CloudFormation::Stack Wed Aug 07 2019 16:42:26 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS nhb-dev-20190802090817 AWS::CloudFormation::Stack Wed Aug 07 2019 16:42:27 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_IN_PROGRESS                           apinhb                 AWS::CloudFormation::Stack Wed Aug 07 2019 16:42:28 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_COMPLETE                              apinhb                 AWS::CloudFormation::Stack Wed Aug 07 2019 16:42:29 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_IN_PROGRESS                           functionnhbusersfunc   AWS::CloudFormation::Stack Wed Aug 07 2019 16:42:29 GMT+0000 (Coordinated Universal Time)
2019-08-07T16:42:30.492Z [INFO]: 
2019-08-07T16:42:30.494Z [INFO]: UPDATE_ROLLBACK_COMPLETE nhb-dev-20190802090817-functionnhbusersfunc-1L6QD27RBKVX AWS::CloudFormation::Stack Wed Aug 07 2019 16:42:30 GMT+0000 (Coordinated Universal Time)
2019-08-07T16:42:40.438Z [INFO]: 
2019-08-07T16:42:40.440Z [INFO]: UPDATE_COMPLETE functionnhbusersfunc AWS::CloudFormation::Stack Wed Aug 07 2019 16:42:39 GMT+0000 (Coordinated Universal Time)
2019-08-07T16:42:45.441Z [INFO]: 
2019-08-07T16:42:45.444Z [INFO]: UPDATE_IN_PROGRESS       authnhbcognito         AWS::CloudFormation::Stack Wed Aug 07 2019 16:42:40 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_COMPLETE          authnhbcognito         AWS::CloudFormation::Stack Wed Aug 07 2019 16:42:41 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_ROLLBACK_COMPLETE nhb-dev-20190802090817 AWS::CloudFormation::Stack Wed Aug 07 2019 16:42:41 GMT+0000 (Coordinated Universal Time)
2019-08-07T16:42:55.505Z [WARNING]: Error updating cloudformation stack
2019-08-07T16:43:00.514Z [INFO]: Following resources failed
2019-08-07T16:43:00.517Z [INFO]: init failed
2019-08-07T16:43:00.518Z [WARNING]: ✖ An error occurred when pushing the resources to the cloud
                                    ✖ There was an error initializing your environment.
2019-08-07T16:43:00.520Z [INFO]: { ResourceNotReady: Resource is not in the state stackUpdateComplete
                                 at constructor.setError (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/resource_waiter.js:182:47)
                                 at Request.CHECK_ACCEPTORS (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/resource_waiter.js:44:12)
                                 at Request.callListeners (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
                                 at Request.emit (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
                                 at Request.emit (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:683:14)
                                 at Request.transition (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:22:10)
                                 at AcceptorStateMachine.runTo (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/state_machine.js:14:12)
                                 at /root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/state_machine.js:26:10
                                 at Request.<anonymous> (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:38:9)
                                 at Request.<anonymous> (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:685:12)
                                 message: 'Resource is not in the state stackUpdateComplete',
                                 code: 'ResourceNotReady',
                                 retryable: false,
                                 time: 2019-08-07T16:42:55.504Z,
                                 statusCode: 200,
                                 retryDelay: 30000 }
2019-08-07T16:43:00.598Z [ERROR]: !!! Build failed
2019-08-07T16:43:00.599Z [ERROR]: !!! Non-Zero Exit Code detected
2019-08-07T16:43:00.600Z [INFO]: # Starting environment caching...
2019-08-07T16:43:00.600Z [INFO]: # Environment caching completed
Terminating logging...
```

**Comment by garyleefight** [comment]
@monaye Amplify CLI released a new version recently. Could you try to install the latest version in pre-build stage.
`        pre_build:
            commands:
                - 'npm install -g @aws-amplify/cli@1.10.0'`

**Comment by monaye** [comment]
@garyleefight 
1.10.0 is the version I am using. 
```
mowin-9m:~ mowin$ amplify -v
1.10.0
```

**Comment by monaye** [comment]
@garyleefight 
no~. still same. 
```
                                 # Starting phase: preBuild
                                 # Executing command: npm install -g @aws-amplify/cli@1.10.0
2019-08-07T19:46:24.483Z [WARNING]: npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
2019-08-07T19:46:31.668Z [WARNING]: npm WARN deprecated left-pad@1.3.0: use String.prototype.padStart()
2019-08-07T19:46:36.637Z [WARNING]: npm WARN deprecated kleur@2.0.2: Please upgrade to kleur@3 or migrate to 'ansi-colors' if you prefer the old syntax. Visit <https://github.com/lukeed/kleur/releases/tag/v3.0.0\> for migration path(s).
2019-08-07T19:46:40.580Z [WARNING]: npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
2019-08-07T19:46:54.500Z [INFO]: /root/.nvm/versions/node/v10.16.0/bin/amplify -> /root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/bin/amplify
2019-08-07T19:46:54.964Z [WARNING]: npm WARN graphql-toolkit@0.5.0 requires a peer of graphql-tag-pluck@^0.8.3 but none is installed. You must install peer dependencies yourself.
2019-08-07T19:46:54.964Z [WARNING]: npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/@aws-amplify/cli/node_modules/fsevents):
                                    npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
2019-08-07T19:46:54.964Z [WARNING]: 
2019-08-07T19:46:54.966Z [INFO]: + @aws-amplify/cli@1.10.0
                                 added 124 packages from 200 contributors, removed 170 packages, updated 141 packages and moved 26 packages in 43.474s
2019-08-07T19:46:54.982Z [INFO]: # Completed phase: preBuild
                                 # Starting phase: build
2019-08-07T19:46:54.983Z [INFO]: # Executing command: amplifyPush --simple
2019-08-07T19:46:54.987Z [INFO]: # Getting Amplify CLI Cloud-Formation stack info from environment cache
2019-08-07T19:46:55.013Z [INFO]: # Start initializing Amplify environment: dev
2019-08-07T19:46:55.013Z [INFO]: # Initializing new Amplify environment: dev (amplify init)
2019-08-07T19:47:01.351Z [INFO]: Note: It is recommended to run this command from the root of your app directory
2019-08-07T19:47:02.738Z [INFO]: 
2019-08-07T19:47:02.739Z [INFO]: For more information on AWS Profiles, see:
                                 https://docs.aws.amazon.com/cli/latest/userguide/cli-multiple-profiles.html
2019-08-07T19:47:02.948Z [WARNING]: - Initializing your environment: dev
2019-08-07T19:47:03.293Z [WARNING]: ✔ Initialized provider successfully.
2019-08-07T19:47:09.156Z [INFO]: Could not read cloudformation template at path: /codebuild/output/src743284947/src/negotiators-handbook/amplify/backend/api/nhb/build/cloudformation-template.json
2019-08-07T19:47:09.204Z [WARNING]: - Updating resources in the cloud. This may take a few minutes...
2019-08-07T19:47:14.799Z [INFO]: 
2019-08-07T19:47:14.802Z [INFO]: UPDATE_IN_PROGRESS nhb-dev-20190802090817 AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:09 GMT+0000 (Coordinated Universal Time) User Initiated
                                 UPDATE_IN_PROGRESS authnhbcognito         AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:14 GMT+0000 (Coordinated Universal Time)
2019-08-07T19:47:19.718Z [INFO]: 
2019-08-07T19:47:19.720Z [INFO]: UPDATE_COMPLETE    authnhbcognito       AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:15 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_IN_PROGRESS functionnhbusersfunc AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:18 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_IN_PROGRESS apinhb               AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:18 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_FAILED      apinhb               AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:18 GMT+0000 (Coordinated Universal Time) Parameters: [DynamoDBEnableServerSideEncryption] do not exist in the template
2019-08-07T19:47:19.885Z [INFO]: 
2019-08-07T19:47:19.885Z [INFO]: UPDATE_IN_PROGRESS          nhb-dev-20190802090817-functionnhbusersfunc-1L6QD27RBKVX AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:18 GMT+0000 (Coordinated Universal Time) User Initiated
                                 UPDATE_ROLLBACK_IN_PROGRESS nhb-dev-20190802090817-functionnhbusersfunc-1L6QD27RBKVX AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:19 GMT+0000 (Coordinated Universal Time) Initiated by parent stack
2019-08-07T19:47:34.723Z [INFO]: 
2019-08-07T19:47:34.725Z [INFO]: UPDATE_FAILED               functionnhbusersfunc   AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:30 GMT+0000 (Coordinated Universal Time) Resource update cancelled
                                 UPDATE_ROLLBACK_IN_PROGRESS nhb-dev-20190802090817 AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:30 GMT+0000 (Coordinated Universal Time) The following resource(s) failed to update: [functionnhbusersfunc, apinhb].
2019-08-07T19:47:39.744Z [INFO]: 
2019-08-07T19:47:39.746Z [INFO]: UPDATE_IN_PROGRESS authnhbcognito       AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:36 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_COMPLETE    authnhbcognito       AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:36 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_IN_PROGRESS functionnhbusersfunc AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:37 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_COMPLETE    apinhb               AWS::CloudFormation::Stack Wed Aug 07 2019 19:47:37 GMT+0000 (Coordinated Universal Time)
2019-08-07T19:48:09.942Z [INFO]: 
2019-08-07T19:48:09.944Z [INFO]: UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS nhb-dev-20190802090817-functionnhbusersfunc-1L6QD27RBKVX AWS::CloudFormation::Stack Wed Aug 07 2019 19:48:07 GMT+0000 (Coordinated Universal Time)
2019-08-07T19:48:14.750Z [INFO]: 
2019-08-07T19:48:14.752Z [INFO]: UPDATE_COMPLETE                              functionnhbusersfunc   AWS::CloudFormation::Stack Wed Aug 07 2019 19:48:11 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS nhb-dev-20190802090817 AWS::CloudFormation::Stack Wed Aug 07 2019 19:48:13 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_IN_PROGRESS                           functionnhbusersfunc   AWS::CloudFormation::Stack Wed Aug 07 2019 19:48:14 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_IN_PROGRESS                           apinhb                 AWS::CloudFormation::Stack Wed Aug 07 2019 19:48:14 GMT+0000 (Coordinated Universal Time)
2019-08-07T19:48:14.954Z [INFO]: 
2019-08-07T19:48:14.955Z [INFO]: UPDATE_COMPLETE apinhb AWS::CloudFormation::Stack Wed Aug 07 2019 19:48:14 GMT+0000 (Coordinated Universal Time)
2019-08-07T19:48:19.957Z [INFO]: 
2019-08-07T19:48:19.959Z [INFO]: UPDATE_ROLLBACK_COMPLETE nhb-dev-20190802090817-functionnhbusersfunc-1L6QD27RBKVX AWS::CloudFormation::Stack Wed Aug 07 2019 19:48:15 GMT+0000 (Coordinated Universal Time)
2019-08-07T19:48:29.775Z [INFO]: 
2019-08-07T19:48:29.777Z [INFO]: UPDATE_COMPLETE          functionnhbusersfunc   AWS::CloudFormation::Stack Wed Aug 07 2019 19:48:25 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_IN_PROGRESS       authnhbcognito         AWS::CloudFormation::Stack Wed Aug 07 2019 19:48:25 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_COMPLETE          authnhbcognito         AWS::CloudFormation::Stack Wed Aug 07 2019 19:48:26 GMT+0000 (Coordinated Universal Time)
                                 UPDATE_ROLLBACK_COMPLETE nhb-dev-20190802090817 AWS::CloudFormation::Stack Wed Aug 07 2019 19:48:26 GMT+0000 (Coordinated Universal Time)
2019-08-07T19:48:39.842Z [WARNING]: Error updating cloudformation stack
2019-08-07T19:48:44.851Z [INFO]: Following resources failed
2019-08-07T19:48:44.854Z [INFO]: init failed
2019-08-07T19:48:44.855Z [WARNING]: ✖ An error occurred when pushing the resources to the cloud
                                    ✖ There was an error initializing your environment.
2019-08-07T19:48:44.858Z [INFO]: { ResourceNotReady: Resource is not in the state stackUpdateComplete
                                 at constructor.setError (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/resource_waiter.js:182:47)
                                 at Request.CHECK_ACCEPTORS (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/resource_waiter.js:44:12)
                                 at Request.callListeners (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
                                 at Request.emit (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
                                 at Request.emit (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:683:14)
                                 at Request.transition (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:22:10)
                                 at AcceptorStateMachine.runTo (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/state_machine.js:14:12)
                                 at /root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/state_machine.js:26:10
                                 at Request.<anonymous> (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:38:9)
                                 at Request.<anonymous> (/root/.nvm/versions/node/v10.16.0/lib/node_modules/@aws-amplify/cli/node_modules/aws-sdk/lib/request.js:685:12)
                                 message: 'Resource is not in the state stackUpdateComplete',
                                 code: 'ResourceNotReady',
                                 retryable: false,
                                 time: 2019-08-07T19:48:39.842Z,
                                 statusCode: 200,
                                 retryDelay: 30000 }
2019-08-07T19:48:44.912Z [ERROR]: !!! Build failed
2019-08-07T19:48:45.055Z [ERROR]: !!! Non-Zero Exit Code detected
2019-08-07T19:48:45.055Z [INFO]: # Starting environment caching...
2019-08-07T19:48:45.055Z [INFO]: # Environment caching completed
Terminating logging...
```

**Comment by garyleefight** [comment]
@monaye Can you provide the local CLI version?

**Comment by monaye** [comment]
@garyleefight My local CLI version is also 1.11.0 
The workaround on this had been introduced by Vinicius C. from AWS support. 
Since I was having an issue with existing project option A worked for me.  

> Option A - in your existing project:
1 - Execute “amplify function add” with Hello World configuration and no access to other services;
2 - Execute "amplify push";
3 - Execute “amplify function remove” removing all related files to backend (option Y);
4 - Execute “amplify push”.

> Option B - in a new project:
1 - Create a new project by running amplify init in a new folder;
2 - Once the init succeeds, copy everything in the amplify/#current-cloud-backend of your current project to backend of the new project you just created;
3 - Open amplify/backend/amplify-meta.json of the newly created project copy the json data under the key providers from amplify/#current-cloud-backend/amplify-meta.json from the newly created project and paste it on to amplify/backend /amplify-meta.jsons providers section
4 - Do an amplify push to update your CloudFormationStack.


> For both options now you would be able to see the tag "DynamoDBEnableServerSideEncryption": "false" in your "amplify\backend\awscloudformation\nested-cloudformation-stack.yml" file. This error was caused by an old version of Amplify CLI that didn’t use this parameter to create the CloudFormationTemplate, which would cause the deployment to fail.

**Comment by monaye** [comment]
The suggested workaround works for first push but failed on second push and I had to change as follow to work : 
```
-    "DynamoDBEnableServerSideEncryption": "false",
+    "DynamoDBEnableServerSideEncryption": false,
```

**Comment by blazinaj** [comment]
Having this same issue. Would this be fixed on the Amplify Console side by first checking if "DynamoDBEnableServerSideEncryption" exists in the template instead of just throwing a build error?

**Comment by blazinaj** [comment]
> The suggested workaround works for first push but failed on second push and I had to change as follow to work :
> 
> ```
> -    "DynamoDBEnableServerSideEncryption": "false",
> +    "DynamoDBEnableServerSideEncryption": false,
> ```

Where in the Cloudformation template does that flag get put in? The function push method is not working for me, I want to try manually adding it

**Comment by heitorlessa** [comment]
I've tried that fix and didn't work for me. 

Here's what I tried:

A. Add function, then remove into an existing project
B. Manually add `"DynamoDBEnableServerSideEncryption": "false",` into `nested-cloudformation-stack.yml`

While B looks like the workaround, it does get removed when I run `amplify push`

Amplify CLI version: 3.0.0

**Comment by heitorlessa** [comment]
Got it working now and there was no need to add/remove function, but to change two files - Here's a quick summary.

1. **Update API nested stack within the backend CloudFormation stack with `DynamoDBEnableServerSideEncryption: true`**

In my case, my API is named `apiawsserverlessairline` - Excerpt from: **amplify/backend/awscloudformation/nested-cloudformation-stack.yml**

```json
"apiawsserverlessairline": {
    "Type": "AWS::CloudFormation::Stack",
    "Properties": {
        "TemplateURL": "https://s3.amazonaws.com/awsserverlessairline-20190506121509-deployment/amplify-cfn-templates/api/cloudformation-template.json",
        "Parameters": {
            "AppSyncApiName": "awsserverlessairline",
            "DynamoDBBillingMode": "PAY_PER_REQUEST",
            "DynamoDBEnableServerSideEncryption": "true", <---
            ...
        }
    }
}
```

2. **Update API parameters.json file with `DynamoDBEnableServerSideEncryption: true`**

Excerpt from: **amplify/backend/api/awsserverlessairline/parameters.json**

```json
{
    "AppSyncApiName": "awsserverlessairline",
    "DynamoDBBillingMode": "PAY_PER_REQUEST",
    "DynamoDBEnableServerSideEncryption": "true", <---
    "AuthCognitoUserPoolId": {
        "Fn::GetAtt": [
            "authcognitoa7d29d9d",
            "Outputs.UserPoolId"
        ]
    }
}
```

**Comment by roland-batovski** [comment]
I am running into the same issue right now. Updated my schema and the `amplify push` now fails every time with:

`UPDATE_FAILED               Answer                                               AWS::CloudFormation::Stack Fri May 22 2020 09:28:17 GMT+0100 (British Summer Time) Parameters: [CreateAPIKey, DynamoDBEnableServerSideEncryption] do not exist in the template`

This error message is thrown for every single one of my models. I tried updating Amplify to the latest version, with no success.
I checked the build folder for the required values in `cloudformation-template.json` and I was able to find them both:

`"DynamoDBEnableServerSideEncryption": {
            "Type": "String",
            "Description": "Enable server side encryption powered by KMS.",
            "Default": "true",
            "AllowedValues": [
                "true",
                "false"
            ]
 }`

`"CreateAPIKey": {
            "Type": "Number",
            "Description": "The boolean value to control if an API Key will be created or not. The value of the property is automatically set by the CLI. If the value is set to 0 no API Key will be created.",
            "Default": 0,
            "MinValue": 0,
            "MaxValue": 1
}`

One thing that I noticed is that I couldn't find any of the above values in the `amplify/backend/awscloudformation/nested-cloudformation-stack.yml` file

I've already tried the fixes suggested above, with no luck. Any insight into this would be appreciated as it's currently stopping us from pushing any updates to our back-end

Cheers!
